package.json

```diff
@@ -13,16 +13,16 @@
     "url": "https://github.com/algolia/autocomplete.js.git"
   },
   "devDependencies": {
-    "angular": "^1.5.8",
-    "angular-mocks": "^1.5.8",
+    "angular": "^1.6.2",
+    "angular-mocks": "^1.6.2",
     "babel-eslint": "^7.0.0",
     "chai": "^3.5.0",
     "colors": "^1.1.2",
     "conventional-changelog-cli": "^1.2.0",
     "doctoc": "^1.2.0",
-    "eslint": "1.5.1",
```

Comment:
Hi, could you update only the angular deps? eslint &co may not be compatible with the code base right now. Thanks